src/dsp_tools/resources/start-stack/start-stack-config.yml

```diff
@@ -1,4 +1,4 @@
 ---
 
 # take commit hash of latest DSP-API release from https://github.com/dasch-swiss/dsp-api/commits/main
-DSP-API commit: 268055603642ada6212c0bfae3ef94a3efea6573
+DSP-API commit: 88b278ebf96a98ddeec5e084745cf31216244368
```

Comment:
The comment says "commit hash of latest DSP-API release". What you took is the commit hash of the latest commit (at the time you created this PR), which is not correct. It's probably okay for this time, but I wanted to mention it for the future :-)

Do you think the comment should be improved?